lib/worker/tests/api/http.rs

```diff
@@ -229,8 +229,8 @@ async fn write_event() {
     app.create_vault_with_cache("api.test", 10).await;
 
     // creating random event
-    let mut event_content = [0u8; 256];
-    thread_rng().fill(&mut event_content);
+    let range = Uniform::from(0..20);
```

Comment:
changed this test so i could replicate the problem. if i do `take(110000000)` I generate a random payload > 100MiB and with that was able to replicate the problem. I kept the value at `100` so the test runs faster